benchmark/Microsoft.IdentityModel.Benchmarks/identitymodel.benchmarks.yml

```diff
@@ -5,6 +5,7 @@ jobs:
   benchmarks:
     source:
       repository: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet
+      branchOrCommit: dev7
```

Comment:
we'll need to remember to go through things and update them to dev after we merge back.